config.sh

```diff
@@ -30,7 +30,7 @@ case `uname` in
 	exit -1
 esac
 
-GITREPO=${GITREPO:-"git://github.com/kaiostech/manifests"}
+GITREPO=${GITREPO:-"https://github.com/kaiostech/manifests.git"}
```

Comment:
Thanks for the contribution!

I don't think you have to add `.git` . We didn't in Capyloon's fork at https://github.com/capyloon/B2G/blob/master/config.sh#L33 and that works fine.

Reply:
For the git clone purpose, either with or without .git will work.